### DISCUSSION TITLE
An old Austrian castle

### DISCUSSION THREAD

**Discussion by rowild** [mkkellogg/GaussianSplats3D#45] [Show and tell]
Thank you for opening the `Discussion` tab!
I take the chance to showcase a splat of mine, so something is posted here. I hope this encourages other people to post something here, too!

Old Austrian castle `Burg Gallenstein` ( [Google Maps](https://www.google.com/maps/place/Burgruine+Gallenstein/@47.6903839,14.6267967,15z/data=!4m2!3m1!1s0x0:0x8b824d60005c57bd?sa=X&ved=2ahUKEwi55de587KCAxVHQvEDHRyhBo0Q_BJ6BAhNEAA) ):
https://stackblitz.com/edit/github-ugddhz?file=src%2Fapp.js

The workflow was:
- recorded with a drone
- converted to a `.ply` with `Luma.ai`
- cleaned-up with `Super Splash`
- converted to a `.splat` with `GaussianSplats3D`
- used `GaussianSplats3D` to bring it to the web

Along the way the orientation became a bit of a hassle (the model was upside down), but with the help of @mkkellogg I could adjust the camera up vector. Thank you, @mkkellogg, for this fantastic script and you great support! Looking forward to what comes next!

I think the best thing with Gaussian Splats and NeRFs and all those new technologies and softwares is that we have a good reason to stand up from our chairs and leave our computer in order to go outside, while still have the feeling to "do" something.

**Comment by mkkellogg** [comment]
Glad you got it worked out, your model looks great!
